src/utils/url.ts

```diff
@@ -1,7 +1,12 @@
-const prefixUrlWithHttp = (url: string) => {
-  if (url.startsWith('http')) return url;
+const prefixUrlWithHttps = (url: string) => {
+  if (url.startsWith('https://')) return url;
 
-  return `http://${url}`;
+  if (url.startsWith('http://')) {
+    const httpsUrl = url.replace('http://', 'https://');
+    return httpsUrl;
+  }
+
+  return `https://${url}`;
 };
```

Comment:
Could also do with a regex like

```suggestion
const prefixUrlWithHttps = (url: string) => 
    url.replace(/(https?:\/\/)?(.+)/, 'https://$2');
```

But depends how regex-averse yall feel 😄

Reply:
🤢

Reply:
Just kidding, I'm fine with either approach

Reply:
Yes, I try to avoid regexes as much as possible 😅 .
But I went for the regex since the name of the function should be descriptive enough.